builtin/providers/aws/resource_aws_autoscaling_group.go

```diff
@@ -95,6 +95,13 @@ func resourceAwsAutoscalingGroup() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"placement_group": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Set:      schema.HashString,
+			},
+
```

Comment:
I just think the addition to the schema should be really simple:

TypeString
Optional

This goes as per the fact the [docs](http://docs.aws.amazon.com/cli/latest/reference/autoscaling/create-auto-scaling-group.html) says that the input is a string and can be empty